### PR TITLE
update sync option settings and added kiln option

### PR DESCRIPTION
**Pull request by DockBoss**
* updated the prompt to say `snap sync` instead of saying `fast sync` on the networks that have snap sync enabled by default. 
* added option for kiln network
### Questions
I didn't add an option for the aa configs, but I can if its needed.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ const options = [{
   pageSize: 10,
   name: 'config',
   message: 'Select network',
-  choices: ['Ethereum (mainnet)', 'Goerli (light Clique testnet)', 'Ropsten (PoS testnet)', 'Sepolia (PoW testnet)', 'Rinkeby (heavy Clique testnet)', 'Gnosis (formerly xDai chain)', 'POA Core (POA mainnet)', 'Spaceneth (local developer node)'],
+  choices: ['Ethereum (PoW mainnet)', 'Goerli (light Clique testnet)', 'Ropsten (PoS testnet)', 'Sepolia (PoS testnet)', 'Kiln (PoS testnet)', 'Rinkeby (heavy Clique testnet)', 'Gnosis (formerly xDai chain)', 'POA Core (POA mainnet)', 'Spaceneth (local developer node)'],
```
Comment (LukaszRozmej) [inline index.js]: everyone knows mainnet consensus, not sure if worth mentioning

Comment (DockBoss) [reply index.js]: I was wondering that. The only reason I added it was to eventually change it to `PoS` after the merge. I can remove it if needed. I'll need to change one or two other lines if I do.

**Comment by gat786** [comment]
Looks good to me

File: index.js (hunk 1 above)
Comment (LukaszRozmej) [reply index.js]: lets remove it

Comment (DockBoss) [reply index.js]: It's updated

**Pull request closed without merging**